### PR TITLE
Added style for institution-list page

### PR DESCRIPTION
**Pull request by kumkum671**
# Checklist
- [X] This PR targets the `main` branch. 
- [X] The commit message is written in past tense.
- [ ] I have added or updated relevant **tests**.
- [ ] I have added or updated relevant **docs**, including release notes if applicable.
- [X] For UI changes, I have attached **screenshots**

image

### CODE REVIEW

File: src/static/styles/central_admin/institution_list/style.scss [hunk 1]
```diff
@@ -1 +1,16 @@
-@import '../../sidebar_navbar_base';
\ No newline at end of file
+@import '../../sidebar_navbar_base';
+
+.wrap{
+    display:flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    .card-wrap{
+        max-width:18rem;
```
Comment (joisemp) [inline src/static/styles/central_admin/institution_list/style.scss:8]: When screen size max width is 767px give max-width 100%

image

**Review by joisemp — CHANGES_REQUESTED** [review]
Make the changes mentioned

File: src/static/styles/central_admin/institution_list/style.scss (hunk 1 above)
Comment (kumkum671) [reply src/static/styles/central_admin/institution_list/style.scss:8]: resolved issue

**Comment by kumkum671** [comment]
solved the card issue  [here](https://github.com/joisemp/sfs-busnest/pull/20/commits/0632a57f394643270420e2690306215d3b0f7266)

image

File: src/static/styles/central_admin/institution_list/style.scss (hunk 1 above)
Comment (joisemp) [reply src/static/styles/central_admin/institution_list/style.scss:8]: Perfect

**Review by joisemp — APPROVED** [review]
Good to merge. However, may need few code formatting later.

**Pull request merged**